docs/gds-supported-platform/helm-chart-best-practices.md

```diff
@@ -0,0 +1,92 @@
+# Helm Chart Best Practices
+
+## Overview
+
+Guidance for authoring Helm chart's suitable for production deployments.
+
+### Use the release name in resource names
+
+`.metadata.name` should include at least `{{ .Release.Name }}`
```

Comment:
we should also probably give some advice about setting Release.Name

Reply:
Probably a bit too much of a bike shedy topic to give advice on really ... Release name could be used for anything from commit SHAs, release number and chart versions to more logical names like prod/staging or even just one offs like we do for "gsp" ... So I'd shy away :smile:

Reply:
I mean, from experience, we've been bitten before by the release name being a long string that just identifies the app and the environment (which was too long) without it being a version.